app/controllers/individual/IndividualAddressController.scala

```diff
@@ -49,18 +49,26 @@ class IndividualAddressController @Inject()(override val messagesApi: MessagesAp
 
   private def actionUrl(id: Int, mode: Mode) = routes.IndividualAddressController.onSubmit(id, mode).url
 
-  implicit val alternativeText: String = "the individual's"
+  implicit val alternativeText: String = "the individual’s"
 
   def onPageLoad(id: Int, mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData).async {
     implicit request =>
 
       val countries = countryListFactory.getCountryList().getOrElse(throw new Exception("Cannot retrieve country list"))
       val form = formProvider(countries)
 
-      val preparedForm = request.userAnswers.get(IndividualAddressPage, id) match {
-        case None => form
-        case Some(value) => form.fill(value)
-      }
+      val preparedForm =
```

Comment:
I'd like to suggest a new apply method to Individual (and organisation) companion objects taking Option(postCode) as a parameter.  Common code could be in the `WithIndividualOrOrganisation` trait.